overcast.py

```diff
@@ -7,51 +7,105 @@
 import requests
 import lxml.html
 import urlparse
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
I think you shouldn't need this here - setting up logging in `overcast-sonos.py` ought to do the trick.